src/App.js

```diff
@@ -2,24 +2,57 @@ import React from 'react';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+import { Clock } from './components/Clock';
+
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 666,
+  }
+
+  visibleTrue = () => {
+    this.setState({
+      isClockVisible: true,
+    });
+  }
+
+  visibleFalse = () => {
+    this.setState({
+      isClockVisible: false,
+    });
+  }
```

Comment:
try to use better names for methods, should be something like `setVisibleTrue` or `showClock`